cmd/optimizely/testdata/default.yaml

```diff
@@ -65,6 +63,8 @@ client:
         redis: 
           host: "localhost:6379"
           password: ""
+          timeout: 5
```

Comment:
5s ?

Reply:
`5s` was possible if the key was at the root and viper was unmarshalling it. In this case we are unmarshalling it at runtime using native `json` library which does not support converting `5s` to time duration.